test/src/kanalytics.spec.js

```diff
@@ -1,5 +1,5 @@
 //eslint-disable-next-line no-unused-vars
-import KAnalyticsPlugin from '../../src/kanalytics'
+import KAnalyticsPlugin from '../../src'
```

Comment:
is it related to the change? how does it know which module from 'src' dir?

Reply:
index.js is the default import path, unless you specify an explicit file name